README.md

```diff
@@ -72,7 +72,92 @@ You can generate it using `BundleAnalyzerPlugin` with `generateStatsFile` option
 command: `webpack --profile --json > stats.json`
 
 `webpack-bundle-analyzer --help` will show you all usage information.
- 
+
+## Contributing
+
+To contribute to `webpack-bundle-analyzer`, fork the repository and clone it to your machine. [See this GitHub help page for what forking and cloning means](https://help.github.com/articles/fork-a-repo/)
+
+### Setup packages
+
+Then install [`yarn`](https://yarnpkg.com/):
+
+```sh
+npm install --global yarn
+```
+
+Next, install this package's dependencies with `yarn`:
+
+```sh
+yarn install --pure-lockfile
+```
+
+We're using `--pure-lockfile` for this first time as the `yarn.lock` file should not be unnecessarily changed unless we modify `package.json`.
+
+### Develop with your own project
+
+Run the following to build this library and watch its source files for changes:
+
+```sh
+yarn run start
+```
+
+You will now have a fully functioning local build of this library ready to be used. **Leave the `start` script running**, and continue with a new Terminal/shell window.
+
+Link the local package with `yarn` and/or `npm` to use it in your own projects:
+
+```sh
+# Needed if your own project uses `yarn` to handle dependencies:
+yarn link
+# Needed if your own project uses `npm` to handle dependencies:
+npm link
```

Comment:
I feel like it's more clear to run the link first in the currently edited package project file and then in your own project. I've tend to have a bad time figuring out the one-liner for `npm link` and come to understand how linking works when I use it this way.

I can of course change this to `npm link ../path/to/your/project` if you feel its better.